app/controllers/auth_controller.rb

```diff
@@ -0,0 +1,63 @@
+class AuthController < Devise::OmniauthCallbacksController
```

Comment:
Can you add a comment at the top explaining what this is for, and mentioning the ENV var that controls it?

Reply:
Done!